build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
     }
 
     dependencies {
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.10"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.31"
```

Comment:
why downgrade?

Reply:
Gradle 7.0.2 includes kotlin 1.4.31 and if we have it at 1.5.10 it's complaining that `Some runtime JAR files in the classpath have an incompatible version`